python/pylibcugraph/CMakeLists.txt

```diff
@@ -64,7 +64,7 @@ if (NOT cugraph_FOUND)
   if(CUGRAPH_BUILD_WHEELS)
     # Statically link dependencies if building wheels
     set(CUDA_STATIC_RUNTIME ON)
-    set(USE_RAFT_STATIC ON)
+    set(USE_RAFT_STATIC OFF)
```

Comment:
Is this okay to do?  I thought DLFW builds (or something) needed this.

Reply:
ok thanks. I just reverted it in https://github.com/rapidsai/cugraph/pull/3167/commits/e51e86830df64ee0b56ef00622966d9a7aafb31b